src/org/exist/launcher/ServiceManager.java

```diff
@@ -42,7 +43,7 @@
 
 public class ServiceManager {
 
-    private final static Logger LOG = Logger.getLogger(ServiceManager.class);
+    private final static Logger LOG = LogManager.getLogger(ServiceManager.class);
```

Comment:
Though this change is good, I think that in .classpath the file is ` lib/user/log4j-1.2-api-2.8.2.jar` is missing. Maybe more?

Reply:
I would ideally suggest not adding the legacy Jar to the `.classpath`. It is not needed for building and may only confuse people by suggesting to import legacy packages when doing auto-complete etc.